src/atoms/Layout/layout.module.scss

```diff
@@ -40,6 +40,16 @@ $border: 1px solid color('neutral-6');
   }
 }
 
+@media #{$small-only} {
+  .padding-small {
```

Comment:
I think it would be helpful to add a little comment here & where it's used. Eventually I would like to see this deprecated and a more robust way to handle these types of use cases.